packages/react-dev-utils/htmlEsmodulesPlugin.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
```

Comment:
It doesn't look like this is used anywhere as you've moved into the `react-scripts` package?

Reply:
Using it for local debugging sorry will delete

Reply:
This should exist in react-dev-utils and be published as a package I assume? So I deleted the other one and kept this one. using yarn link for debugging now

Reply:
Seems reasonable. It's inline with other plugins we have created like `InlineChunkHtmlPlugin`.